### PR TITLE
+clu #17870 add Java api for joinSeedNodes

### PR DESCRIPTION
**Pull request by ktoso**
Resolves https://github.com/akka/akka/issues/17870

### CODE REVIEW

File: akka-cluster/src/test/java/akka/cluster/ClusterJavaCompileTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@SuppressWarnings("ConstantConditions")
+public class ClusterJavaCompileTest {
+
+  final ActorSystem system = null;
+  final Cluster cluster = null;
+
+
+  public void compileJoinSeedNodesInJava() {
+    final List<Address> addresses = new ArrayList<>();
+    addresses.add(new Address("akka.tcp", "MySystem"));
```
Comment (patriknw) [inline akka-cluster/src/test/java/akka/cluster/ClusterJavaCompileTest.java]: I'd use `Arrays.asList(` or `Collections.singletonList`

**Comment by patriknw** [comment]
LGTM

File: akka-cluster/src/test/java/akka/cluster/ClusterJavaCompileTest.java (hunk 1 above)
Comment (ktoso) [inline akka-cluster/src/test/java/akka/cluster/ClusterJavaCompileTest.java]: good catch thanks

**Comment by ktoso** [comment]
Failure was unrelated

**Pull request merged**